syncer/filter.go

```diff
@@ -24,85 +24,28 @@ import (
 	onlineddl "github.com/pingcap/dm/syncer/online-ddl-tools"
 )
 
-// skipQuery will return true when
-// - given `sql` matches builtin pattern.
-// - any schema of table names is system schema.
-// - any table name doesn't pass block-allow list.
-// - type of SQL doesn't pass binlog filter.
-func (s *Syncer) skipQuery(tables []*filter.Table, stmt ast.StmtNode, sql string) (bool, error) {
+func (s *Syncer) filterQueryEvent(tables []*filter.Table, stmt ast.StmtNode, sql string) (bool, error) {
 	if utils.IsBuildInSkipDDL(sql) {
```

Comment:
is this check duplicated with `filterSQL`, can we remove one  ?

Reply:
Because `filterSQL` and `filterQueryEvent` are not serial execution after the https://github.com/pingcap/dm/pull/2090. `filterSQL` will only be called when SQL parse failed. So I think needn't remove it.